types/types.proto

```diff
@@ -234,14 +234,138 @@ message VMRemoveResponse {
   string cause  = 2;
 }
 
+message UserContainerPort {
+  int32 hostPort      = 1;
+  int32 containerPort = 2;
+  int32 servicePort   = 3;
+  string protocol     = 4;
+}
+
+message UserVolumeReference {
+  string path   = 1;
+  string volume = 2;
+  bool readOnly = 3;
+}
+
+message UserFileReference {
+  string path     = 1;
+  string filename = 2;
+  string perm     = 3;
+  string user     = 4;
+  string group    = 5;
+}
+
+message UserContainer {
+  string name                           = 1;
+  string image                          = 2;
```

Comment:
in hypervisor/pod.go, there is `User          UserUser` field here.
we will have to add User as `UserUser user = 13` ?